src/tools/_tools.ms.scss

```diff
@@ -0,0 +1,9 @@
+// /*------------------------------------*\
+//     #MS
+// \*------------------------------------*/
+
+// To clearly indicate that a scale value is used, use it with a custom
+// function which just passes the number through.
+@function ms($scale: 0) {
+    @return $scale;
```

Comment:
Add a check to only allow configured scales, otherwise return `@error`

Reply:
Done.